foursight_core/react/api/react_routes.py

```diff
@@ -166,14 +184,17 @@ def reactapi_route_checks_history_latest(env: str, check: str) -> Response:  # n
     @route("/{env}/checks/{check}/history/{uuid}", authorize=True)
     def reactapi_route_checks_history_uuid(env: str, check: str, uuid: str) -> Response:  # noqa: implicit @staticmethod via @route
         """
-        Returns the result of the given check.
+        Returns the result of the given check result by its (name and) uuid.
         """
         return app.core.reactapi_checks_history_uuid(app.current_request.to_dict(), env, check=check, uuid=uuid)
 
     @route("/{env}/checks/history/recent", authorize=True)
     def reactapi_route_checks_history_recent(env: str) -> Response:  # noqa: implicit @staticmethod via @route
         """
-        Returns all recent check run history, across all checks.
+        Returns all "recent" check run history, across all checks.
+        By default this return (at most) the 25 most recent results;
+        a limit argument may be specified to get a different number.
+        Paging is NOT currently supported.
```

Comment:
Meaning there's an arg protocol for paging that is ignored? Or that there is no protocol for that? Or there's a protocol but you'll get an error for using it?

Reply:
Meaning no protocol support. I clarified the comments on this.